models/order_detailed_cluster.py

```diff
@@ -2,12 +2,13 @@
 from kmodes.kmodes import KModes
 
 def model(dbt, fal):
-    dbt.config({
-        # Look at `fal_project.yml` to see packages installed with this environment
-        "fal_environment": "clustering"
-    })
+    # Look at `fal_project.yml` to see packages installed with this environment
+    dbt.config(fal_environment="clustering")
```

Comment:
hmm, is this the right syntax?

Reply:
Yes, it runs with that and then without it:

```
14:12:53  11 of 11 START python table model dbt_fal.orders_forecast ...................... [RUN]
14:12:54  Unhandled error while executing target/run/jaffle_shop/models/orders_forecast.py
No module named 'prophet'
14:12:54  11 of 11 ERROR creating python table model dbt_fal.orders_forecast ............. [ERROR in 0.94s]
14:12:54
14:12:54  Finished running 4 view models, 7 table models in 0 hours 0 minutes and 6.53 seconds (6.53s).
14:12:54
14:12:54  Completed with 1 error and 0 warnings:
14:12:54
14:12:54  No module named 'prophet'
```